### DISCUSSION TITLE
How to monitor opentelemetry-java-agent?

### DISCUSSION THREAD

**Discussion by nobeh** [open-telemetry/opentelemetry-java-instrumentation#8399] [Q&A]
Hi there,

Are there any metrics that are exposed/published from opentelemetry-java-agent itself? The goal is to be able to monitor the agent itself at runtime and its possible side effects on resource consumption next to the running JVM.

Thanks

**Comment by srinusanduri** [comment]
Hi @nobeh , did you find the agent monitoring metrics? I am trying to setup agent monitoring, could you share your inputs if you have setup already. I have started a similar conversation here https://github.com/open-telemetry/opentelemetry-java-instrumentation/discussions/10284#discussioncomment-8362573.
